lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -8,7 +8,9 @@
          cdkTrapFocusAutoCapture>
         <ng-content select="adf-viewer-toolbar"></ng-content>
         <ng-container *ngIf="showToolbar && !toolbar">
-            <adf-toolbar id="adf-viewer-toolbar" class="adf-viewer-toolbar">
+                  <adf-toolbar id="adf-viewer-toolbar" class="adf-viewer-toolbar" 
+                  [ngClass]="{ 'adf-viewer-right-close-button': closeButtonPosition === 'right'}"
```

Comment:
Is there any need for this class ?? I don't see it being used anywhere

Reply:
Removed